### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by andrewcr7**
*Issue #, if available:*

*Description of changes:*
Fixed typo

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by andrewcr7** [comment]
Typo in doc 'CreateReplicationGrooup' - 'CreateReplicationGroup'

File: doc_source/redis/backups-restoring.md [hunk 1]
```diff
@@ -126,4 +126,4 @@ For more information, see the following:
 + [Creating a Redis Replication Group from Scratch](Replication.CreatingReplGroup.NoExistingCluster.md) in the *ElastiCache User Guide*\.
 + [CreateReplicationGroup](https://docs.aws.amazon.com/AmazonElastiCache/latest/APIReference/API_CreateReplicationGroup.html) in the *ElastiCache API Reference*\.
 
- 
\ No newline at end of file
+
```
Comment (phansys) [inline doc_source/redis/backups-restoring.md:129]: IMO, this change is not related.

**Review by phansys — APPROVED** [review]

**Review by spaethp — APPROVED** [review]
